contents/articles/2017-03_hololens-bt300-addon/index.md

```diff
@@ -0,0 +1,60 @@
+--- 
+title: Hololens and BT300 VR/AR Addon
+date: Thu Mar 02 2017 15:28:39 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "We are happy to announce that we have added the Microsoft Hololens and Epson Moverio BT-300 VR/AR addons, now available on the store!"
+featured_img: "../../../../media/images/blog/addon_hololens-dark.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/addon_hololens.jpg"
+---
+
+<img src="../../../../media/images/blog/addon_hololens-light.jpg" class='Feature-image u-padBottom--2' alt="Hololens Addon">
+
+We are happy to announce that we have added the Epson Moverio BT-300 and Microsoft Hololens VR/AR addons, now available on the store! 
+
+You can easily augment these VR/AR hardware with Pupil Labs eye trackers that can run up to 120hz and have a latency of 5.7ms, without compromising your VR or AR experience. Fast enough for interaction applications and detection of saccades and quick fixations. And With our [open source software](https://github.com/pupil-labs/pupil/releases/latest) and direct access to the raw eye video feeds you are never locked into a platform.
```

Comment:
You can easily add eye tracking functionality to the Hololens and the BT300 - no tools required! Our cameras run at 120hz and give you access to a low latency eye tracking pipeline. Fast enough for interaction... (continue with the of this sentence here). 

You get direct access to raw eye video feeds and all data from our software. We have open source building blocks that you can use to integrate eye tracking into your application (link to hmd-eyes)